src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -1,20 +1,25 @@
 package core.basesyntax;
 
 public class FootballMatch {
+     public static void main(String[] args) {
+        System.out.println(matchResult(1,2,1,1));
+    }
```

Comment:
don't create main in solutions. you can use it for tests, but don't forget to remove before commit